doc/api-details.md

```diff
@@ -27,8 +27,8 @@ If a user wants to change any of the above variables, they should be added to th
 
 ### Create an admin user
```

Comment:
I think this whole section is already covered on the `getting-started` page https://github.com/gctucker/kernelci-api/blob/doc-admin-user/doc/getting-started.md#create-an-admin-user-account.
We can drop this here to avoid duplication.

Reply:
Well, maybe we'll need to update the API documentation. But I think everything about the API should be covered in the API documentation, whereas the "Getting Started" page provides step-by-step instructions to get a system working. So some things might be explained twice, with a focus on a particular use-case in the "Getting Started" section.